FoodBookApp/UI/Views/Settings/SettingsView.swift

```diff
@@ -0,0 +1,70 @@
+//
+//  SettingsView.swift
+//  FoodBookApp
+//
+//  Created by Laura Restrepo on 11/05/24.
+//
+
+import Foundation
+import SwiftUI
+
+struct SettingsView: View {
+    @State private var daysSinceLastReview = UserDefaults.standard.bool(forKey: "sendDaysSinceLastReviewNotification")
+    @State private var days = UserDefaults.standard.integer(forKey: "daysSinceLastReview") - 1
+    @State private var reviewsUploaded = UserDefaults.standard.bool(forKey: "sendReviewsUploadedNotification")
+    @State private var lunchTime = UserDefaults.standard.bool(forKey: "sendLunchTimeNotification")
+    
+    let customGray2 = Color(red: 242/255, green: 242/255, blue: 247/255)
+    
+    var body: some View {
+        VStack(alignment: .leading) {
+            // Sheet rectangle
+            Rectangle()
+                .fill(Color.gray)
+                .frame(width: 40, height: 5)
+                .cornerRadius(3)
+                .padding(.top, 10)
+                .frame(maxWidth: .infinity, alignment: .center)
+            
+            // Title
+            Text("Settings")
+                .fontWeight(.bold)
+                .frame(maxWidth: .infinity, alignment: .center)
+            
+            Form {
+                Section(header: Text("NOTIFICATIONS")) {
+                    Toggle("Days since last review", isOn: $daysSinceLastReview)
+                        .onChange(of: daysSinceLastReview) {
+                            UserDefaults.standard.set(daysSinceLastReview ? true : false, forKey: "sendDaysSinceLastReviewNotification")
+                        }
+                    if daysSinceLastReview {
+                        Picker("Number of days", selection: $days) {
+                            ForEach(1..<8) { day in
+                                Text("\(day) days")
+                            }
+                        }.onChange(of: days) {
+                            UserDefaults.standard.set(days + 1, forKey: "daysSinceLastReview")
+                        }
+                    }
+                    Toggle("Reviews uploaded", isOn: $reviewsUploaded)
+                        .onChange(of: reviewsUploaded) {
+                            UserDefaults.standard.set(reviewsUploaded ? true : false, forKey: "sendReviewsUploadedNotification")
+                        }
+                    Toggle("Lunch time", isOn: $lunchTime)
+                        .onChange(of: lunchTime) {
+                            UserDefaults.standard.set(lunchTime ? true : false, forKey: "sendLunchTimeNotification")
+                        }
+                }
+                // TODO: @maria
```

Comment:
Your turn @mar-cas3 

<img alt="the crown" height="150" src="https://media.vanityfair.com/photos/5a2f53ca577ffb294a25322e/1:1/w_2692,h_2692,c_limit/the-crown-season-3.jpg">